src/marvin/beta/assistants/runs.py

```diff
@@ -200,7 +200,9 @@ async def run_async(self) -> "Run":
             handler = event_handler_class(**self.event_handler_kwargs)
 
             try:
-                self.assistant.pre_run_hook()
+                self.assistant.pre_run_hook(
+                    assistant=self.assistant, run=self, run_kwargs=run_kwargs
```

Comment:
@zzstoatzz Should it just pass `run=self`? Reason being, method will have access to assistant.self already and run_kwargs can be derived from self.

Reply:
oh yeah, good point